src/providers.ts

```diff
@@ -439,6 +442,9 @@ export function getFeatureRegEx(reg:string): RegExp {
 	if (reg == 'getFeatureVariableJSON') {
 		return REGEX_J
 	}
+	if (reg == 'getAllFeatureVariables') {
```

Comment:
the regex is used for the second parameter of feature calls.  There is no second parameter on getAllFeatureVariables.